metrics/prometheus/prometheus_test.go

```diff
@@ -156,6 +156,54 @@ func TestRequestMetric(t *testing.T) {
 		})
 }
 
+func TestDebugRequestMetric(t *testing.T) {
+	testCases := []struct {
+		description                      string
+		givenDebugEnabledFlag            bool
+		givenAccountDebugMetricsDisabled bool
+		expectedAccountDebugCount        float64
+		expectedDebugCount               float64
+	}{
+		{
+			description:                      "Boolean flags should result in both metrics being updated",
```

Comment:
Please specify what the boolean flags are set to in the description. For readability purposes and for when tests fail these descriptions help us easily understand what was passed in and what was expected. I like the descriptions you have for the last two test cases. Could you please update these to have the same format as well? 

I noticed the same thing in the `go_metrics_test.go` file as well